tools/chunk/src/main/scala/org/allenai/nlpstack/chunk/OpenNlpChunker.scala

```diff
@@ -15,21 +18,23 @@ class OpenNlpChunker extends Chunker {
       if (t.string == "-") PostaggedToken(t, ":") else t
     }
 
-    val chunks = chunker.chunk(tokens.map(_.string).toArray, fixedTokens.map(_.postag).toArray)
+    val chunks = chunker_t.get().chunk(tokens.map(_.string).toArray, fixedTokens.map(_.postag)
+      .toArray)
     (tokens zip chunks) map { case (token, chunk) => ChunkedToken(token, chunk) }
   }
 
   //Open NLP runs into random NullPointerExceptions. The following is a workaround that reduces the
   // probability of running into errors
-  def chunkPostagged(tokens: Seq[PostaggedToken]): Seq[ChunkedToken] = {
-    def helper(tries: Int): Seq[ChunkedToken] = try {
-      if (tries == 0) throw java.lang.NullPointerException
-      else chunkPostaggedOriginal(tokens)
-    } catch {
-      case e: Throwable => helper(tries - 1)
-    }
-    helper(3)
-  }
+//  def chunkPostagged(tokens: Seq[PostaggedToken]): Seq[ChunkedToken] = {
+//    def helper(tries: Int): Seq[ChunkedToken] = try {
+//      if (tries == 0) throw new java.lang.NullPointerException()
+//      else chunkPostaggedOriginal(tokens)
+//    } catch {
+//      case e: Throwable => helper(tries - 1)
+//    }
+//    helper(0)
+//  }
+  def chunkPostagged(tokens: Seq[PostaggedToken]): Seq[ChunkedToken] = chunkPostaggedOriginal(tokens)
```

Comment:
These are all remnants from your first attempt at fixing this, right? I don't think we need `chunkPostaggedOriginal` anymore. Everything can just go into `chunkPostagged`. And the commented code should go as well.